docs/Deploying/Configuration.md

```diff
@@ -2117,6 +2117,21 @@ Maximum age of a vulnerability (a CVE) to be considered "recent". The age is cal
        recent_vulnerability_max_age: 48h
   ```
 
+### disable_win_os_vulnerabilities 
```

Comment:
Let's also document the osquery version necessary for this feature.

Reply:
https://github.com/fleetdm/fleet/issues/7415